localstack/services/s3/s3_starter.py

```diff
@@ -308,7 +308,7 @@ def s3_response_is_delete_keys(self, request, path, bucket_name):
             is_delete_keys_v3 = (
                 query and ("delete" in query) and get_safe(query, "$.x-id.0") == "DeleteObjects"
             )
-            return is_delete_keys_v3 or is_delete_keys(request, path, bucket_name)
+            return is_delete_keys_v3 or is_delete_keys(request, path)
```

Comment:
Thanks for this fix @viren-nadkarni ! 👍 (this was an oversight after the last moto upgrade). Do you think we could add a small test that invokes this part of the code, to avoid regressions in the future?

Reply:
Actually, let's pick this up in a follow-up PR - going to merge now, as this is a critical fix blocking some customers.. 👍